app/logic/Calendar/OWA/OWACalendar.ts

```diff
@@ -26,6 +26,10 @@ export class OWACalendar extends Calendar {
     return this.mainAccount as OWAAccount;
   }
 
+  get mailbox(): string | undefined {
```

Comment:
What is this? Why it is needed? Why is it called "mailbox" when it returns the username? Can you please rename it to reflect its meaning? Like e.g.


```suggestion
  /** If this is a calendar of another user, return the username of that other user account.
   * In the normal case, when this is the calendar of our user, returns undefined. */
  get sharedAccountUsername(): string | undefined {
```